src/pages/WorldsDashboard/WorldsDashboard.scss

```diff
@@ -51,4 +51,21 @@
   &__messages-container {
     text-align: center;
   }
+
+  &__worlds-list {
+    background-color: $dark;
+
+    display: grid;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: max-content;
+    grid-gap: 1rem;
+  }
+
+  &__world-panel {
+    border: 1px solid $white;
```

Comment:
I think `$white` is (to be) removed. Ideally there would be `$content--border` but I guess `$content--over` might be enough